src/text-editor.coffee

```diff
@@ -1561,6 +1561,16 @@ class TextEditor extends Model
   setCursorBufferPosition: (position, options) ->
     @moveCursors (cursor) -> cursor.setBufferPosition(position, options)
 
+  # Essential: Get a {Cursor} at given screen coordinates {Point}
+  #
+  # * `position` A {Point} or {Array} of `[row, column]`
+  #
+  # Returns the first matched {Cursor} or undefined
+  getCursorAtScreenPosition: (position) ->
+    for cursor in @cursors
+      return cursor if cursor.getScreenPosition().isEqual(position)
+    undefined
```

Comment:
@maxbrunsfeld that's interesting about coffee-script using a `for` as a value if it's the last line - i'm pretty new to coffee-script so I had no idea about that. 

The reason I was considering a `return` was from the contribution documentation:
>Using a plain return when returning explicitly at the end of a function.
>Not return null, return undefined, null, or undefined

Not sure how I should proceed here, `undefined` or `null`.